src/Renderer/Label2DRenderer.js

```diff
@@ -174,10 +181,10 @@ class Label2DRenderer {
 
             this.showNodeDOM(object);
 
-            object.children.forEach(c => this.culling(c, currentMaxZoom, extent));
+            object.children.forEach(c => this.culling(c, extent));
         // By verifying the maxzoom and the presence of the label inside the
         // visible extent, we can filter more labels.
```

Comment:
This comment could be updated, since the `maxzoom` is no longer verified

Reply:
yes